blog/2025/terraform-provider-release-5.adoc

```diff
@@ -0,0 +1,48 @@
+:title: Keycloak Terraform Provider Release 5
+:date: 2025-01-10
+:publish: false
```

Comment:
```suggestion
:publish: true
```

I assume you want it to be visible the moment you merge the PR. I know last time we had that problem, and therefore I suggest to have it marked as "true" all the time.